emuDCS/PeripheralCore/include/emu/pc/TMB_constants.h

```diff
@@ -321,10 +323,10 @@ static const unsigned long int  v6_gtx_rx3_adr          = 0x000152;  //ADR_V6_GT
 static const unsigned long int  v6_gtx_rx4_adr          = 0x000154;  //ADR_V6_GTX_RX4
 static const unsigned long int  v6_gtx_rx5_adr          = 0x000156;  //ADR_V6_GTX_RX5
 static const unsigned long int  v6_gtx_rx6_adr          = 0x000158;  //ADR_V6_GTX_RX6
-
-// extra DCFEB Bad Bits on OTMB 
```

Comment:
I think we need these comments kept in place.
I see that a fraction of the code around here is moved for no particular reason (other than the comments removal, extra spaces here and there). It's not so relevant for the head version, but is important to track-back the history of changes to the file ("blame" feature)